File: yongjunleeme/numeric_strings_and_english_words.java
class Solution {
    public int solution(String s) {
        String arr[] = {"zero","one","two","three","four","five","six","seven","eight","nine"};
        for(int i = 0; i < 10; i++)
            s = s.replace(arr[i], Integer.toString(i));
        return Integer.parseInt(s);
    }
}

File: yongjunleeme/numeric_strings_and_english_words.cpp
#include <string>
#include <vector>

using namespace std;

int solution(string s) {
    string arr[10] = {"zero","one","two","three","four","five","six","seven","eight","nine"};
    for(int i = 0; i < 10; i++){
        while(1){
            auto it = s.find(arr[i]);
            if(it == string::npos) break;
            s.replace(it, arr[i].size(), to_string(i));
        }
    }
    return stoi(s);
}
